src/main/java/picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.java

```diff
@@ -311,5 +343,27 @@ private static void trackOpticalDuplicates(final List<? extends ReadEnds> list,
         if (opticalDuplicates > 0) {
             opticalDuplicatesByLibraryId.increment(list.get(0).getLibraryId(), opticalDuplicates);
         }
+        return opticalDuplicates;
     }
+
+    private static void trackDuplicateCounts(final int listSize,
+                                                   final int optDupCnt,
```

Comment:
Not sure why this is `optDupCnt` and there is `opticalDuplicatesCountHist` too. Consistent spelling would be nice, e.g. `opticalDuplicatesCount`.